### PR TITLE
Work in Progress on 2022

### PR DESCRIPTION
**Pull request by andyland**

### CODE REVIEW

File: iBurn/src/main/java/com/gaiagps/iburn/service/DataUpdateService.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public static void scheduleAutoUpdate(Context context) {
                 .setUpdateCurrent(true)
                 .build();
 
-        GcmNetworkManager.getInstance(context).schedule(dailyUpdate);
+//        GcmNetworkManager.getInstance(context).schedule(dailyUpdate);
```
Comment (andyland) [inline iBurn/src/main/java/com/gaiagps/iburn/service/DataUpdateService.java:37]: This is deprecated, crashes the app.  Needs to be migrated to WorkManager

File: iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt [hunk 2]
```diff
@@ -301,74 +303,74 @@ class MapboxMapFragment : Fragment() {
...
-                engine.requestLocationUpdates()
-                val plugin = LocationLayerPlugin(mapView, map)
-                plugin.locationEngine = engine
-                plugin
-            } else if (hasLocationPermission) {
-                LocationLayerPlugin(mapView, map)
-            } else {
-                null
-            }
+            val style = Style.Builder().fromUri("asset://map/style.json").withSource(VectorSource("composite", "asset://map/map.mbtiles"))
```
Comment (chrisballinger) [inline iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt:306]: The font glyphs folder is also required or the map won't render. (its within the same folder as the map and style in the data repo)

File: .gitignore [hunk 3]
```diff
@@ -45,3 +45,5 @@ iBurn/output.json
 *.aab
 
 fabric.properties
+iBurn/google-services.json
```
Comment (chrisballinger) [inline .gitignore:48]: I have a Firebase config for us to use for Crashlytics

File: iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt (hunk 2 above)
Comment (andyland) [reply iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt]: Yeah, just checked in

File: iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt [hunk 4]
```diff
@@ -902,7 +914,7 @@ class MapboxMapFragment : Fragment() {
                         mappedItems.add(userPoi)
                         if (marker != null) {
                             mappedCustomMarkerIds[marker.id] = userPoi
-                            callback?.invoke(marker)
+                            // callback?.invoke(marker)TODO
```
Comment (chrisballinger) [inline iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt:917]: What are the repercussions of commenting this out?

File: iBurn/src/main/java/com/gaiagps/iburn/location/LocationProvider.java [hunk 5]
```diff
@@ -196,9 +194,9 @@ public void removeLocationUpdates(LocationEngineCallback<LocationEngineResult> r
...
         }
 
-
-        public Type obtainType() {
-            return Type.MOCK;
-        }
+//
+//        public Type obtainType() {
+//            return Type.MOCK;
+//        }
```
Comment (chrisballinger) [inline iBurn/src/main/java/com/gaiagps/iburn/location/LocationProvider.java:200]: Do we still want this?

**Review by chrisballinger — COMMENTED** [review]
This is awesome! Just a few comments but I think overall this is looking good to merge

**Review by chrisballinger — APPROVED** [review]

File: iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt (hunk 4 above)
Comment (andyland) [reply iBurn/src/main/java/com/gaiagps/iburn/MapboxMapFragment.kt]: Addressed here: https://github.com/iBurnApp/iBurn-Android/pull/13/commits/2c17d4bf1f02c514408d8a2d04f5cd8ff19ce362

**Pull request merged**